automaticRelease.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+file="./gradle.properties"
+
+# Reads a property from a file by grepping and throwing away everything before "="
+function prop {
+    grep "${1}" ${file} | cut -d'=' -f2
+}
+
+version=$(prop 'version')
+release_major=0
+release_minor=0
+release_patch=0
+
+# break down the version number into it's components
+regex="([0-9]+).([0-9]+).([0-9]+)-SNAPSHOT"
```

Comment:
This always expects versions to end with "-SNAPSHOT", is that the intended behaviour?

Reply:
See above, that script was a leftover